server/routes/friendlist/friendList.js

```diff
@@ -0,0 +1,157 @@
+const express = require("express");
+const router = express.Router();
+const { check, validationResult } = require("express-validator");
+const authentication = require("../../middleware/authentication");
+
+//Models
+const FriendList = require("../../models/FriendList");
+const User = require("../../models/User");
+
+/*
+    Type: POST route
+    Desc: Create a new Friends list
+    Acc: private
+    Params: user, friendListname, friends
+*/
+router.post(
+  "/add",
```

Comment:
I think, similar to @aeciorc 's comments on the friends backend, the post route and delete round should have the same route but different verb?
```suggestion
  "/lists",
```